src/widgets/avatar.cpp

```diff
@@ -24,7 +24,7 @@
 #include <QPaintEvent>
 #include <QPainter>
 #include <QDebug>
-
+#include <QPainterPath>
```

Comment:
In #68 I added it under the QPainter include for better ordering, but the whole include stuff are not ordered so... it's okay either way.